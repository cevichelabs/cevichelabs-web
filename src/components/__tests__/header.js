import React from "react"
import renderer from "react-test-renderer"

import Header from "../header"

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Header siteTitle='CevicheLabs Web' menuLinks={
        [
          {
            name: "METODOLOGÍA",
            description: "Cómo lo que hacemos",
            link: '/metodologia',
          },
          {
            name: "SOLUCIONES",
            description: "Qué es lo que hacemos",
            link: '/soluciones',
          },
          {
            name: "PLANES",
            description: "En qué te ayudamos",
            link: '/planes',
          },
          {
            name: "EXPERIENCIA",
            description: "Cómo te ayudamos",
            link: '/experiencia',
          }
        ]
      } />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
