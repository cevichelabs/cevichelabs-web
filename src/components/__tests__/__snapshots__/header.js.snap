// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="sc-bdnylx fjhTox"
>
  <div
    className="sc-dlnjPT hiBepg"
  >
    <div
      className="sc-gtssRu ldLFCG"
    >
      <h1
        className="sc-hKFyIo ewETJm"
      >
        <a
          href="/"
        >
          <img
            alt="CevicheLabs"
            data-src="test-file-stub"
            src="test-file-stub"
          />
        </a>
      </h1>
    </div>
    <nav
      className="sc-gKAblj gvkgBm"
    >
      <a
        href="/metodologia"
      >
        <big
          className="sc-iCoHVE davxlP"
        >
          METODOLOGÍA
        </big>
        <small
          className="sc-fujyUd cxYhpr"
        >
          Cómo lo que hacemos
        </small>
      </a>
      <a
        href="/soluciones"
      >
        <big
          className="sc-iCoHVE davxlP"
        >
          SOLUCIONES
        </big>
        <small
          className="sc-fujyUd cxYhpr"
        >
          Qué es lo que hacemos
        </small>
      </a>
      <a
        href="/planes"
      >
        <big
          className="sc-iCoHVE davxlP"
        >
          PLANES
        </big>
        <small
          className="sc-fujyUd cxYhpr"
        >
          En qué te ayudamos
        </small>
      </a>
      <a
        href="/experiencia"
      >
        <big
          className="sc-iCoHVE davxlP"
        >
          EXPERIENCIA
        </big>
        <small
          className="sc-fujyUd cxYhpr"
        >
          Cómo te ayudamos
        </small>
      </a>
    </nav>
    <h2
      className="sc-eCApGN iUguQn"
    >
      <span
        className="sc-jSFkmK fwaloA"
      >
        .
      </span>
      Laboratorio de soluciones digitales
    </h2>
    <div
      className="sc-pNWxx cbPmqA"
      onClick={[Function]}
    />
  </div>
</header>
`;
